ui-components/src/locales/general.json

```diff
@@ -1467,6 +1479,7 @@
         "household": "Household",
         "age": "Age",
         "disability": "Disability",
+        "accessibility": "Accessibility Features",
```

Comment:
This makes the progress nav a little cramped, maybe just "Accessibility"?

Reply:
fixed